### DISCUSSION TITLE
Identifying unmapped reads in RNAseq and BS DNA data

### DISCUSSION THREAD

**Discussion by sr320** [sr320/ceabigr#11] [Q&A]
There are a good deal of sequence reads that do not map to the genome. For various reasons it would be nice to know what those are. One might be if we see presence of consistent secondary taxa, that could be integrated into our analysis and serve as a proxy for data such as microbiome or symbionts. We have the `fq` files of unmapped reads. What are some robust approaches to identify these reads?

**Comment by kubu4** [comment]
[MEGAN6](https://software-ab.informatik.uni-tuebingen.de/download/megan6/welcome.html) is designed to assign taxa at the read level, so is well suited for that aspect of the task.

**Reply by kubu4** [reply]
I used MEGAN6. If people are interested in checking out the results, see [this issue](https://github.com/sr320/ceabigr/issues/22). TL;DR - Taxonomic assignment of unmapped BSseq reads were primarily (i.e. > 75% of all reads in all samples) assigned to _Crassostrea virginica_.
